es-app/src/main.cpp

```diff
@@ -196,6 +197,28 @@ bool verifyHomeFolderExists()
 	return true;
 }
 
+void locales_init() {
+#ifdef HAVE_GETTEXT
+  char* btd;
+  char* cs;
+
+#define LOCALEDIR INSTALLATION_PREFIX_DIR "/share/locale"
+// local dir ; uncomment if you want to use locally compiled .mo files instead of .mo from the installation dir (present only after make install)
+//#define LOCALEDIR "./locale/lang"
```

Comment:
RetroPie doesn't rely on `make install` for installation.  Can we make `./locale/lang` the enabled option?